process/sig-lifecycle.md

```diff
@@ -0,0 +1,13 @@
+# Special Interest Group (SIG) Life Cycle
+
+Special Interest Groups are under the direct governance of a Working Group (WG) which can create and close them as necessary and appropriate. SIGs are expected to have a very specific goal and not necessarily live for a very long time.
```

Comment:
Suggest "specific _and terminable_ goal".

Reply:
As above, not clear why needs to be terminable.

Reply:
+1